modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_permissions.jsp

```diff
@@ -289,6 +285,12 @@ PortletConfigurationPermissionPropagation portletConfigurationPermissionPropagat
 	);
 
 	if (<portlet:namespace />permissionPropagationEnabledCheckbox) {
+		Liferay.Util.SessionStorage.setItem(
+			'<portlet:namespace />count',
+			0,
+			Liferay.Util.SessionStorage.TYPES.PERSONALIZATION
+		);
```

Comment:
Resetting the change count value here doesn't make sense to me. It should be handled in the modal instead of here.

Reply:
Fixed it. Thanks!